content/docs/setup/kubernetes/multicluster-install/index.md

```diff
@@ -349,3 +350,106 @@ rules to reach the proper Istio service in the main cluster.
 Within this option there are 2 sub-options.  One is to re-use the default Istio ingress gateway
 installed with the provided manifests or helm charts.  The other option is to create another
 Istio ingress gateway specifically for multicluster.
+
+## Security
+
+Istio supports deployment of mTLS between the control plane components as well as between
+sidecar injected application pods.
+
+### Control plane security
+
+The steps to enable control plane security are as follows:
+
+1.  Istio control plane cluster deployed with
+    1.  control plane security enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+
+1.  Istio remote clusters deployed with
+    1.  control plane security enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+        1.  The CA certificate for the remote clusters needs to be signed by the same CA or root CA as the main cluster.
+    1.  Istio pilot service hostname resolvable via DNS
+        1.  Required because Istio configures the sidecar to verify the certificate subject names using the `istio-pilot.<namespace>` subject name format.
+    1.  Control plane IPs or resolvable host names set
+
+### mTLS between application pods
+
+The steps to enable mTLS for all application pods are as follows:
+
+1.  Istio control plane cluster deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+
+1.  Istio remote clusters deployed with
+    1.  Global mTLS enabled
+    1.  `citadel` certificate self signing disabled
+    1.  a secret named `cacerts` in the Istio control plane namespace with the [CA certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key)
+        1.  The CA certificate for the remote clusters needs to be signed by the same CA or root CA as the main cluster.
+
+**NOTE:**  The CA certificate steps are identical for both control plane security and application pod security steps.
```

Comment:
please use > (block indent) rather than note.

Reply:
Ok ... Martin had complaints about overuse of block indent so I was hesitant to use it.